packages/ios_platform_images/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -49,26 +50,27 @@
 		0EF1CD9A3A3064B5289EF22E /* Pods-Runner.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Runner.release.xcconfig"; path = "Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"; sourceTree = "<group>"; };
 		1498D2321E8E86230040F4C2 /* GeneratedPluginRegistrant.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = GeneratedPluginRegistrant.h; sourceTree = "<group>"; };
 		1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GeneratedPluginRegistrant.m; sourceTree = "<group>"; };
-		33B0E32B2AB348FF0006F0C1 /* UIImage+ios_platform_imagesTests.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = "UIImage+ios_platform_imagesTests.m"; sourceTree = "<group>"; };
+		3131C9A22ADEFDA8000703FA /* UIImage+ios_platform_imagesTests.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; name = "UIImage+ios_platform_imagesTests.m"; path = "../../../../../../../Downloads/UIImage+ios_platform_imagesTests.m"; sourceTree = "<group>"; };
```

Comment:
You'll need to revert all of the project changes related to this file; you've pointed it to a location outside of the repo on your local machine.

Reply:
Good observation! 
This probably happens when you drag files into xcode, and picked "reference only" instead of "copy files"